src/main/angular/model/organization.ts

```diff
@@ -19,6 +19,10 @@ export class Organization implements IAMResource {
     return this.identification.name;
   }
 
+  containsAsterisk(): boolean {
```

Comment:
generally this information would be better encoded as metadata, rather than encoded as part of the name. 

Here's a practical example where this design would fail. Back in my college days, I was in charge of running a digital collaboration space for several ACM clubs - we called it ACM*. In this case, this method would interpret the org name as missing, even though the name was there. :(

However, because this is an edge case and we're running low on time, I'll approve.